analyze_company_ques_networking.ipynb

```diff
@@ -114,7 +114,7 @@
     "    os.chdir(folder)\n",
     "    print(os.getcwd(), '\\n Now importing model\\n')\n",
     "\n",
-    "    model = BERTopic.load('model_ngram_no_len')     # default stored using 'model_cpu'\n",
+    "    model = BERTopic.load('networking_ngrm_trained_model_no_len')     # default stored using 'model_cpu'\n",
```

Comment:
Can you change the model name as previous one or whatever is available in the folder

Reply:
Yes, I have updated the name